histree-frontend/tsconfig.json

```diff
@@ -1,31 +1,26 @@
 {
-	"extends": "@tsconfig/svelte/tsconfig.json",
-	"compilerOptions": {
-		"target": "ESNext",
-		"useDefineForClassFields": true,
-		"module": "ESNext",
-		"resolveJsonModule": true,
-		"baseUrl": ".",
-		/**
+  "extends": "@tsconfig/svelte/tsconfig.json",
+  "compilerOptions": {
+    "target": "ESNext",
+    "useDefineForClassFields": true,
+    "module": "ESNext",
+    "resolveJsonModule": true,
+    "baseUrl": ".",
+    /**
      * Typecheck JS in `.svelte` and `.js` files by default.
      * Disable checkJs if you'd like to use dynamic types in JS.
      * Note that setting allowJs false does not prevent the use
      * of JS in `.svelte` files.
      */
-		"allowJs": true,
-		"checkJs": true,
-		"isolatedModules": true,
-		"strict": true
-	},
-	"include": [
-		"src/**/*.d.ts",
-		"src/**/*.ts",
-		"src/**/*.js",
-		"src/**/*.svelte"
-	],
-	"references": [
-		{
-			"path": "./tsconfig.node.json"
-		}
-	]
-}
\ No newline at end of file
+    "allowJs": true,
+    "checkJs": true,
+    "isolatedModules": true
+    // "strict": true
```

Comment:
I tried pulling and uncommenting this and it seemed to still pass style checks. Do you mind trying to change this on your branch and let me know if it works for you?